### ISSUE TITLE
sms:<name> doesn't sanitize name for SQL query

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#141]
```
Example:

(4:11:23 PM) admin@coq.dk/Pidgin99362300: sms:a'b
(4:11:24 PM) phone@coq.dk: Error: sms (a'b) near "b": syntax error: , while 
compiling: SELECT number FROM alias WHERE aliasName='a'b'
(4:11:30 PM) admin@coq.dk/Pidgin99362300: sms:a'b:hello
(4:11:31 PM) phone@coq.dk: Error: sms (a'b:hello) near "b": syntax error: , 
while compiling: SELECT number FROM alias WHERE aliasName='a'b'


Additional informations:
GTalkSMS version = 2.1.3
Android OS version = 2.2
Phone model = HTC Hero



```

Original issue reported on code.google.com by `Eckankar` on 14 Apr 2011 at 2:12

**Comment by GoogleCodeExporter** [comment]
```
Pfeeeh, those damn Apostrophes. Thanks for reporting!
```

Original comment by `fschm...@gmail.com` on 14 Apr 2011 at 4:19
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.2
```

Original comment by `Florent....@gmail.com` on 24 Apr 2011 at 1:16
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:50
- Changed state: **Deployed**
